### PR TITLE
Python lab script

### PR DESCRIPTION
**Pull request by YashTatkondawar**

### CODE REVIEW

File: tests/system/python/packages/test_fog.py [hunk 1]
```diff
@@ -0,0 +1,574 @@
+
```
Comment (praveen-garg) [inline tests/system/python/packages/test_fog.py:1]: Let's rename it to test_lab.

File: tests/system/python/packages/test_fog.py [hunk 2]
```diff
@@ -0,0 +1,574 @@
...
+            time.sleep(1)
+        
+        if LOOP == retries - 1 :
+            print ("TIMEOUT! negative_sine event not fired. No /tmp/out file.")
+        else :
+             print ("---- negative_sine event fired ----") 
+        
+        print ("======================= EVENT NEGATIVE SINE COMPLETE =======================")
+    
+class TestToggledEvent:
```
Comment (praveen-garg) [inline tests/system/python/packages/test_fog.py:522]: move TestToggledEvent (1) test and TestPositiveNegativeSineNotification (2) under TestEventEngine itself. and also add assertions as required.

**Review by praveen-garg — CHANGES_REQUESTED** [review]

**Comment by praveen-garg** [comment]
@YashTatkondawar  Closing this pull request, as it is coming from the forked repo. Please open a new PR

**Comment by YashTatkondawar** [comment]
Okay sir once I update the above mentioned changes I will open a new PR.

**Pull request closed without merging**